### ISSUE TITLE
Combo box for Foreign Keys does not work at Browse Data

### ISSUE THREAD

**Issue by csanyipal** [sqlitebrowser/sqlitebrowser#2104]
Details for the issue
--------------------

#### What did you do?
I click ones then once again. This is not a double click.
When I click the second time on a field which has reference to another table (because it is a foreign key) the combo box is showed for a moment then immediately disappeared so I can't browse the possible values for this field, but sometimes it succeeds so I can browse the possible values for this field. If I do a double click on that field nothing happen, I can't see the combo box even for a second.

#### What did you expect to see?
The combo box so I can select the proper value from the other table.

#### What did you see instead?
I only see it for a second but then the combo box disappeared very quickly.

Useful extra information
-------------------------

The info below often helps, please fill it out if you're able to. :)

#### What operating system are you using?
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.3 LTS
Release:	18.04
Codename:	bionic

#### What is your DB4S version?
Version 3.11.99 (Nov 16 2019)
Built for x86_64-little_endian-lp64, running on x86_64
Qt Version 5.5.1
SQLCipher Version 3.2.0 (based on SQLite 3.8.6)

#### Did you also

- [No ] Try out the latest nightly version: https://github.com/sqlitebrowser/sqlitebrowser#nightly-builds
- [ ] Search for an existing similar issue: https://github.com/sqlitebrowser/sqlitebrowser/issues?utf8=%E2%9C%93&q=is%3Aissue%20
I find this:
https://github.com/sqlitebrowser/sqlitebrowser/issues/192

**Comment by scottfurry** [comment]
Start `sqlitebrowser` from command line and repeat steps above. Is anything spat back out onto the console when things abort?

**Comment by csanyipal** [comment]
I started sqlitebrowser from the command line and repeat steps above. There is nothing spat back out onto the console when trying to use the combo box on the field.

**Comment by scottfurry** [comment]
@mgrojo, @chrisjlocke - escalating. I'm not sure what else to try here.

**Comment by csanyipal** [comment]
Just to mention this: I realised that that I am using actually the nightly build of DB4S.

**Comment by mgrojo** [comment]
Yes,  3.11.99 is the nightly build.

No idea of what is happening, I cannot reproduce it. Nevertheless, what is surprising is the Qt version that your build is using. It is too old for a nightly version, but I see that it is the one used for our continouous build AppImage, so I guess you are using that.

I've tried to reproduce it using that version too, without result. Maybe a recorded video or gif can give some insight.

As a workaround, you can try using F2.

**Comment by csanyipal** [comment]
Here is the GIF created from video:

![DB4S_ForeignKey_issue](https://user-images.githubusercontent.com/1310341/73217164-af771a80-4157-11ea-877a-d17340193758.gif)

**Comment by csanyipal** [comment]
I think that that this issue could cause a bad mice - the left click of my mice works a little uncertain. I will buy a new mice soon. Then I shell see whether can I reproduce this issue.

**Comment by csanyipal** [comment]
No, it's not the mouse that makes this error. I just tried with the touchpad of my laptop - on which is the Xubuntu operating system. The same issue exists with the touchpad too.

**Comment by heylix** [comment]
For me the combo box doesn't even show up. Xfce 4.14 on Fedora 32.

sqlitebrowser 3.11.2, Qt 5.13.2, SQLite 3.30.1.

![grafik](https://user-images.githubusercontent.com/36211486/88284400-a4a7d780-cced-11ea-87da-c2d3ed778478.png)

No errors when run via console.

**Comment by csanyipal** [comment]
There is another thread for this issue, see #2200 here, on github.

**Comment by heylix** [comment]
I just noticed when I triple-click, a dropdown appears but I can only select the value, I don't have a list of entries. Same with sqlitebrowser 3.12.0 from rawhide.

**Comment by mgrojo** [comment]
#2200  was reported as solved using a nightly. Is it not the same with this?
